src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountPlugin.groovy

```diff
@@ -71,7 +71,7 @@ class DexMethodCountPlugin implements Plugin<Project> {
             return false
         }
 
-        // Java version strings are something like 1.8.0_65; we don't
+        // Java version strings are something like 1.8.0_65, we don't
```

Comment:
TIL.  No big loss, as generally by the time this task is run all other work is finished.